src/main/java/org/jitsi/videobridge/cc/GenericAdaptiveTrackProjectionContext.java

```diff
@@ -16,6 +16,7 @@
 package org.jitsi.videobridge.cc;
 
 import org.jetbrains.annotations.*;
+import org.jitsi.impl.neomedia.portaudio.*;
```

Comment:
I'm wondering what could have caused the IDE to (presumably) automatically pull this package.

Reply:
Probably an autocomplete typo.  Fixed.